$MirOS: ports/print/cups/patches/patch-berkeley_lpr_c,v 1.2 2006/06/21 19:25:33 bsiegert Exp $
--- berkeley/lpr.c.orig	Sat Jul 12 00:48:26 2008
+++ berkeley/lpr.c	Tue Oct 21 22:56:22 2008
@@ -290,7 +290,7 @@ main(int  argc,				/* I - Number of comm
 	      num_copies = atoi(argv[i]);
 	    }
 
-            sprintf(buffer, "%d", num_copies);
+            snprintf(buffer, sizeof(buffer), "%d", num_copies);
             num_options = cupsAddOption("copies", buffer, num_options, &options);
 	    break;
 
